### PR TITLE
[Christmas] Y'all want a Smite Cranberry?

### PR DESCRIPTION
**Pull request by FairlySadPanda**
## About the PR

image

## Why / Balance

image

(Mostly a thank-you to the mods and admins for their work this year)

## Technical details

1. Added Smite Cranberry, including both the can and the re-agent.
2. Renamed Lemon-Lime to Smite to match; there's no other changes.
3. Between one and three Smite Cranberry cans are ejected from a Christmas anomaly on its pulse.
4. Smite Cranberry cans, and only Smite Cranberry cans, have a plunger-like effect when thrown. When fired out of a gun, such as a Christmas anomaly or improvised pneumatic cannon, the cans do 1 blunt and 2 cold damage each. This theoretically allows someone, with a sufficient supply of Smite Cranberry, to create the equivalent of a plunger gun.

## Media

image

Sprite by @august-sun 

Updated anomaly effect:
https://github.com/user-attachments/assets/256939fb-db82-4bee-b532-ccc9a4765d8c

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

**Changelog**

:cl: FairlySadPanda August-Sun Eric156
- add: Smite Cranberry is now available wherever lemon-lime soda is sold, just for the holidays. This includes a certain jolly anomaly.

### CODE REVIEW

**Comment by FairlySadPanda** [comment]
Immediate suggestion from @IProduceWidgets - an alcoholic variant to put in the booze-o-mat.

**Comment by FairlySadPanda** [comment]
It would be good to have some way of marking up items and vending machine contents as seasonal, and being able to hang these features off of a seasonal cvar setting.

**Comment by FairlySadPanda** [comment]
Other name suggestions

1. Slips/Slips Cranberry @august-sun 
2. Smite/Cranberry Smite @Plykiya

**Comment by FairlySadPanda** [comment]
I should like to point out that after my extended break and vow to only review stuff on here for a long while, this stupid meme has brought me back

curse you all

File: Resources/Prototypes/Recipes/Reactions/drinks.yml [hunk 1]
```diff
@@ -1230,4 +1230,4 @@
     ZombieBlood:
       amount: 1
   products:
-    ZombieCocktail: 4
\ No newline at end of file
+    ZombieCocktail: 4
```
Comment (FairlySadPanda) [inline Resources/Prototypes/Recipes/Reactions/drinks.yml:1233]: This added newline is so minor I really don't want to have to faff removing it...

**Comment by beck-thompson** [comment]
(High because it needs to be added before christmas!)

**Comment by august-sun** [comment]
> I should like to point out that after my extended break and vow to only review stuff on here for a long while, this stupid meme has brought me back
> 
> curse you all

that's why we do this 😎

**Comment by Emisse** [comment]
arbitrage fail

**Comment by TeenSarlacc** [comment]
it's the thirst thirstiest time

**Comment by FairlySadPanda** [comment]
OK y'all I may have slightly more to this PR

**Comment by FairlySadPanda** [comment]
https://github.com/user-attachments/assets/e453f898-27e7-4098-baee-4c6bb924b946

**Review by Emisse — APPROVED** [review]

File: Resources/Locale/en-US/reagents/meta/consumable/drink/soda.ftl [hunk 2]
```diff
@@ -22,6 +22,9 @@ reagent-desc-ice-cream = It was either this or the microwave, and nobody wants i
 reagent-name-lemon-lime = lemon-lime
 reagent-desc-lemon-lime = tangy lime and lemon soda
 
+reagent-name-lemon-lime-cranberry = lemon-lime-cranberry
+reagent-desc-lemon-lime-cranberry = tart cranberry, Christmas, and a hint of lemon and lime
```
Comment (slarticodefast) [inline Resources/Locale/en-US/reagents/meta/consumable/drink/soda.ftl:26]:
```suggestion
reagent-desc-lemon-lime-cranberry = Tart cranberry, Christmas, and a hint of lemon and lime.
```

File: Resources/Textures/Objects/Consumable/Drinks/lemon-lime-cranberry.rsi/meta.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    {
+      "name": "icon"
+    },
+	{
+	  "name": "icon_open"
+	},
+    {
+      "name": "inhand-right",
+      "directions": 4,
+      "delays": [
```
Comment (slarticodefast) [inline Resources/Textures/Objects/Consumable/Drinks/lemon-lime-cranberry.rsi/meta.json:19]: Why do the inhands have delays?

**Review by slarticodefast — CHANGES_REQUESTED** [review]

File: Resources/Textures/Objects/Consumable/Drinks/lemon-lime-cranberry.rsi/meta.json (hunk 3 above)
Comment (FairlySadPanda) [reply Resources/Textures/Objects/Consumable/Drinks/lemon-lime-cranberry.rsi/meta.json:19]: Parity with lemon-lime.rsi

Comment (FairlySadPanda) [reply Resources/Textures/Objects/Consumable/Drinks/lemon-lime-cranberry.rsi/meta.json:19]: (they'll never know I copy-pasta'd)

**Review by slarticodefast — APPROVED** [review]

**Pull request merged**